FirebaseAppDistributionInternal.podspec

```diff
@@ -0,0 +1,64 @@
+#
+# Be sure to run `pod lib lint FirebaseAppDistributionInternal.podspec' to ensure this is a
+# valid spec before submitting.
+#
+
+Pod::Spec.new do |s|
+    s.name             = 'FirebaseAppDistributionInternal'
+    s.version          = '10.7.0-beta'
+    s.summary          = 'Firebase App Distribution Internal for Swift implementations'
+
+    s.description      = <<-DESC
+    Not for public use.
+    SDK for Swift code in Firebase App Distribution.
+                         DESC
+
+    s.homepage         = 'https://firebase.google.com'
+    s.license          = { :type => 'Apache-2.0', :file => 'LICENSE' }
+    s.authors          = 'Google, Inc.'
+
+    s.source           = {
+      :git => 'https://github.com/firebase/firebase-ios-sdk.git',
+      :tag => 'CocoaPods-' + s.version.to_s
```

Comment:
This line would need to change to successfully push to SpecsDev

Reply:
Would it make sense to avoid pushing the podspec to master then? And just push it to SpecsDev?

Reply:
I'm fine either way. Note that "push" means two different things. For master it's a git push. For Specs Dev, it's a CocoaPods publish.

Reply:
Makes sense! Reverted the `podspec` from master for now. I'll do a repo push to SpecsDev to fix the issue!